index.html

```diff
@@ -83,19 +93,36 @@ <h1>My Recent Works</h1>
             src="https://res.cloudinary.com/david79/image/upload/v1674029969/work%20stuff/Img-Placeholder_vhaslg.png"
             alt="Image Placeholder"
           />
-          <h3>Multi-Post Stories</h3>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="flex">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-
+          <div class="multi_stories_content">
+              <h3>Multi-Post Stories</h3>
+              <p>
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text ever
+                  since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="flex">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                
+                <button class="btn">See Project</button>
+            </div>
+        </div>
+        <div class="project flex">
+          <div class="project_info flex">
+            <h3>Profesional Art Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="flex">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
```

Comment:
- [ ] Good job creating your cards and adding the required interaction :+1:, but I think it would be better if you set a fixed space between each card because when the page grows the space between them also grows.

**Your design**

![Screenshot from 2023-01-26 07-53-32](https://user-images.githubusercontent.com/105117832/214845390-951ee0ad-496e-4059-8d6c-3910995da386.png)

**Figma**

![Screenshot from 2023-01-26 07-56-08](https://user-images.githubusercontent.com/105117832/214845667-6a2e155e-5d63-416d-8347-db905c3bf5e7.png)